test/controllers/v1/openapi_endpoint_test.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module V1
+  class OpenapiEndpointTest < ActionDispatch::IntegrationTest
+    test 'openapi v1 success' do
+      get '/api/compliance/v1/openapi.json'
```

Comment:
Can we get an assertion before calling `get` on `User.current = nil` ? That's just to make sure this can get called without authorization or any current user set. I think this test could be rendered useless in the future if by mistake we set the User.current somewhere else.